packages/@n8n/benchmark/scripts/n8nSetups/scaling/docker-compose.yml

```diff
@@ -0,0 +1,176 @@
+services:
+  redis:
+    image: redis:6-alpine
```

Comment:
I usually use and see the regular redis image, wondering if it matters that benchmarking will use a more lightweight image?

Reply:
Doesn't really matter, just smaller image to pull it faster